### PR TITLE
chore(test): special handling for iso builds

### PR DESCRIPTION
**Pull request by cbr7**
### What does this PR do?
Adds a special flag to build ISO images and increases timeout in such a case, because ISO builds take much longer than others.

### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop-extension-bootc/issues/662

### CODE REVIEW

File: tests/playwright/src/model/bootc-page.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import type { Locator, Page } from '@playwright/test';
 import { expect as playExpect } from '@playwright/test';
 import { waitUntil } from '@podman-desktop/tests-playwright';
 import { ArchitectureType } from '@podman-desktop/tests-playwright';
+import { time } from 'node:console';
```
Comment (odockal) [inline tests/playwright/src/model/bootc-page.ts:23]: unused import?

Comment (cbr7) [reply tests/playwright/src/model/bootc-page.ts]: yeah, fixed.

**Review by odockal — APPROVED** [review]

File: tests/playwright/src/bootc-extension.spec.ts [hunk 2]
```diff
@@ -124,9 +126,18 @@ describe('BootC Extension', async () => {
         await playExpect.poll(async () => await imagesPage.waitForImageExists(imageName)).toBeTruthy();
       }, 150000);
 
-      test.skipIf(isLinux).each(['QCOW2', 'AMI', 'RAW', 'VMDK', 'ISO'])(
-        `Building bootable image type: %s`,
-        async type => {
+      describe.skipIf(isLinux).each(['QCOW2', 'AMI', 'RAW', 'VMDK', 'ISO'])('Building images ', async type => {
```
Comment (benoitf) [inline tests/playwright/src/bootc-extension.spec.ts:129]: I'm not sure if it's possible but shouldn't it be something like

```ts
const platforms = '['QCOW2', 'AMI', 'RAW', 'VMDK'];
if (!buildISOImage) {
   platforms.push('ISO')
}

 describe.skipIf(isLinux).each(platforms)('Building images ', async type => {
```

Comment (cbr7) [reply tests/playwright/src/bootc-extension.spec.ts:129]: Does that work? I was under the impression that vitest creates the test context before anything else, all the examples for parametrized tests that I've seen have the parameters provided directly to the each() without any previous logic.

Comment (benoitf) [reply tests/playwright/src/bootc-extension.spec.ts:129]: No idea it's why I emitted some doubts :-)

Comment (cbr7) [reply tests/playwright/src/bootc-extension.spec.ts:129]: @benoitf I just tried it, it seems that you can send params to the test just using a variable, however you can't manipulate the number of tests, basically it uses whatever value `platforms` has if defined globally or it reads it undefined and does not account for any changes to it after the test context is created.

So yes it seems that all the test contexts are created before any logic is executed.

Comment (benoitf) [reply tests/playwright/src/bootc-extension.spec.ts:129]: Ok thanks so ignore my comment

**Pull request merged**